openfisca_aotearoa/aotearoa_legislationmodel.py

```diff
@@ -0,0 +1,40 @@
+"""
+This file defines our country's tax and benefit system.
+
+A tax and benefit system is the higher-level instance in OpenFisca.
+Its goal is to model the legislation of Aotearoa.
+Basically a tax and benefit system contains simulation variables (source code) and legislation parameters (data).
+
+See https://openfisca.org/doc/key-concepts/tax_and_benefit_system.html
+"""
+
+import os
+
+from openfisca_core.taxbenefitsystems import TaxBenefitSystem
+
+from openfisca_aotearoa.conf.cache_blacklist import \
+    cache_blacklist as conf_cache_blacklist
+from openfisca_aotearoa.entities import entities
```

Comment:
Backslashes as line continuation aren't widely used in Python
```suggestion
from openfisca_aotearoa.conf.cache_blacklist import (
    cache_blacklist as conf_cache_blacklist,
)
```

Reply:
Ahh yes - resolved as suggested